src/lib/backend.c

```diff
@@ -5,10 +5,27 @@
 #include "backend_esysdb.h"
 #include "backend_fapi.h"
 
-static bool is_fapi_preview_enabled(void) {
+enum backend {
+    backend_error,
+    backend_esysdb,
+    backend_fapi
+};
 
-    const char *env = getenv("FAPI_PREVIEW");
-    return env && ((!strcmp(env, "yes") || !strcmp(env, "true")));
+static enum backend get_backend(void) {
+
+    const char *env = getenv("TPM2_PKCS11_BACKEND");
+
+    if (!env) {
```

Comment:
I don't know why this block of code bothers me so much, but I am going to be pedantic.
```C
typedef enum backend backend;
enum backend {
    backend_error,
    backend_esysdb,
    backend_fapi
};

static backend get_backend(void) {

    const char *env = getenv("TPM2_PKCS11_BACKEND");

    if (!env || !strcasecmp(env, "esysdb")) {
        return backend_esysdb;
    }

    if (!strcasecmp(env, "fapi")) {
        return backend_fapi;
    }

    return backend_error;
}
```

Reply:
done